modules/commons/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Bad naming. `commons` is too generic, while this module only provides some DB script utils

Reply:
Replaced by database-commons.